chef_master/source/_templates/nav-docs.html

```diff
@@ -64,6 +64,18 @@
           {
             "title": "Licensing",
             "hasSubItems": false,
+            "subItems": [
+              {
+                "title": "Accepting the Chef License",
+                "hasSubItems": false,
+                "url": "/chef_license_accept.html"
+              }
+              {
+                "title": "Accepting the Chef License",
```

Comment:
This title is listed twice...is this intentional? :)

Reply:
Am removing this double listing as it is stopping the local build from happening.